airflow/providers/amazon/aws/transfers/s3_to_redshift.py

```diff
@@ -42,7 +42,7 @@ class S3ToRedshiftOperator(BaseOperator):
     :param schema: reference to a specific schema in redshift database
     :param table: reference to a specific table in redshift database
     :param s3_bucket: reference to a specific S3 bucket
-    :param s3_key: reference to a specific S3 key
+    :param s3_key: reference either to a specific S3 key or a set of keys or folders sharing that prefix
```

Comment:
What does “a set of keys” mean? Does it has to be a Python set, or is the term being used more liberally? If the latter case I think _collection_ is a more common term.

Reply:
I suppose a better wording would be:
```
:param s3_key: key prefix that selects single or multiple objects from S3
```